applications/GeoMechanicsApplication/custom_elements/stress_state_policy.h

```diff
@@ -32,7 +32,9 @@ class StressStatePolicy
                                                                  double DetJ,
                                                                  const Geometry<Node>& rGeometry) const = 0;
     [[nodiscard]] virtual Vector CalculateGreenLagrangeStrain(const Matrix& rDeformationGradient) const = 0;
-    [[nodiscard]] virtual std::unique_ptr<StressStatePolicy> Clone() const = 0;
+    [[nodiscard]] virtual std::unique_ptr<StressStatePolicy> Clone() const        = 0;
+    [[nodiscard]] virtual const Vector&                      GetVoigtVector()     = 0;
```

Comment:
I think this can be a const function 
```suggestion
    [[nodiscard]] virtual const Vector&                      GetVoigtVector() const = 0;
```